sample-apps/react-native/dogfood/src/screens/LiveStream/ViewLiveStream.tsx

```diff
@@ -52,6 +110,15 @@ export const ViewLiveStreamWrapper = ({
     getCall();
   }, [call]);
 
+  useEffect(() => {
+    call?.on('error', (e) => {
+      if (e.error && e.error.code !== 104) {
+        return;
+      }
+      console.log('Livestream Call Left');
+    });
+  }, [call]);
```

Comment:
why do we add this hook?

Reply:
Again, I missed removing it. Sorry, will remove it. I was testing automatic call leave on LIVE_END error received.

Reply:
Removed 👍🏻